qutebrowser/browser/qutescheme.py

```diff
@@ -201,6 +203,26 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('tabs')
+def qute_tabs(_url):
+    """Handler for qute://tabs. Display information about all open tabs."""
+    tabs = collections.defaultdict(list)
+    for win_id, window in objreg.window_registry.items():
+        if sip.isdeleted(window):
+            continue
+        tabbed_browser = objreg.get('tabbed-browser',
+                                    scope='window',
+                                    window=win_id)
+        for tab in tabbed_browser.widgets():
+            if tab.url() != QUrl("qute://tabs/"):
+                tabs[str(win_id)].append((tab.title(), tab.url().url()))
```

Comment:
Damn, I missed this one: This should be `tab.url().toDisplayString()` so it strips passwords. If you prefer I can fix it up while merging, though.

Reply:
Fixed it (I should have see that when looking at the doc)